### ISSUE TITLE
dbus-daemon launched but not terminated

### ISSUE THREAD

**Issue by jlesage** [gilbertchen/duplicacy-cli#29]
I use duplicacy on a server running Ubuntu 16.04.  I noticed that there are a couple of hundred of `dbus-daemon` processes on the server:

```
ps -fu root | grep dbus-daemon | grep -v grep
root     32317     1  0 Sep27 ?        00:00:00 /usr/bin/dbus-daemon --fork --print-pid 4 --print-address 6 --session
root     32339     1  0 Nov08 ?        00:00:00 /usr/bin/dbus-daemon --fork --print-pid 4 --print-address 6 --session
root     32414     1  0 Oct08 ?        00:00:00 /usr/bin/dbus-daemon --fork --print-pid 4 --print-address 6 --session
root     32443     1  0 Oct08 ?        00:00:00 /usr/bin/dbus-daemon --fork --print-pid 4 --print-address 6 --session
root     32578     1  0 Oct23 ?        00:00:00 /usr/bin/dbus-daemon --fork --print-pid 4 --print-address 6 --session
[...]
ps -fu root | grep dbus-daemon | grep -v grep | wc -l
773
```

After some investigation, I found that duplicacy-cli is the one that create them (via `dbus-launch`), but never kill them.

To reproduce, you can just run `duplicacy-cli -h` and see that each time there is an additional `dbus-deamon` process.

This causes memory usage of the machine to increase and will eventually prevent any new process to be created.

**Comment by gilbertchen** [comment]
If you run 'export $(dbus-launch)' once prior to running Duplicacy, it will not start a new instance of dbus-daemon every time.  This sets the environment variable DBUS_SESSION_BUS_ADDRESS so that duplicacy knows how to communicate with the existing dbus instance.

This is just a workaround.  If DBUS_SESSION_BUS_ADDRESS is not set, Duplicacy shouldn't start a new dbus instance (which is used for saving/retrieving passwords).  I'll try to fix this in the next update.

**Comment by gilbertchen** [comment]
This should have been fixed in version 1.1.8.

**Comment by jlesage** [comment]
Yes, looks good.  Thanks!
